processes/ingest/publisher_backlist.py

```diff
@@ -10,24 +10,43 @@ class PublisherBacklistProcess(CoreProcess):
     def __init__(self, *args):
         super(PublisherBacklistProcess, self).__init__(*args[:4])
 
-        self.ingest_offset = int(args[5] or 0)
-        self.ingest_limit = (int(args[4]) + self.ingestOffset) if args[4] else 5000
-        self.full_import = self.process == 'complete' 
-
-        self.generateEngine()
-        self.createSession()
+        self.limit = (len(args) >= 5 and args[4] and args(4) <= 100) or None
+        self.offset = (len(args) >= 6 and args[5]) or None
 
         self.s3_bucket = os.environ['FILE_BUCKET']
         self.createS3Client()
 
+        self.publisher_backlist_service = PublisherBacklistService()
+        
     def runProcess(self):
         try:
-
-            response = airtable_integration.create_airtable_request()
+            self.generateEngine()
+            self.createSession()
+
+            if self.process == 'daily':
+                records = self.publisher_backlist_service.get_records(offset=self.offset, limit=self.limit)
+            elif self.process == 'complete':
+                records = self.publisher_backlist_service.get_records(full_import=True)
+            elif self.process == 'custom':
+                records = self.publisher_backlist_service.get_records(start_timestamp=self.ingestPeriod, offset=self.offset, limit=self.limit)
+            else: 
+                logger.warning(f'Unknown Publisher Backlist ingestion process type {self.process}')
+                return
+            
+            raise Exception
+        
+            for record in records:
+                self.addDCDWToUpdateList(record)
             
-            print(response)
+            self.saveRecords()
+            self.commitChanges()
+
+            logger.info(f'Ingested {len(self.records)} Publisher Backlist records')
 
         except Exception as e:
-            logger.exception('Failed to run Pub Backlist process')
-            raise e
+            logger.exception('Failed to run Publisher Backlist process')
+            raise e   
+        finally:
+            self.close_connection()
+
     
```

Comment:
There may be a \t character here - recommend removing it so Github stops complaining.